src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/AbstractFactory.php

```diff
@@ -30,6 +30,7 @@ abstract class AbstractFactory implements SecurityFactoryInterface
         'check_path' => '/login_check',
         'use_forward' => false,
         'require_previous_session' => false,
+        'login_path' => '/login',
```

Comment:
Is this change still needed?

Reply:
Yes, we use `array_intersect_key()` between all config options of the user and this array. The resulting array is passed to `FormLoginAuthenticator`. It needs to know the login_path (for `getLoginPath()`)

Reply:
Please note that this line changes nothing for the user (the option existed for the success/failure handlers before), it only changes what is passed to the authenticator.